library/src/main/java/com/schibsted/spain/barista/assertion/BaristaImageViewAssertions.kt

```diff
@@ -2,16 +2,25 @@ package com.schibsted.spain.barista.assertion
 
 import android.support.annotation.DrawableRes
 import android.support.annotation.IdRes
-import android.support.test.espresso.Espresso
-import android.support.test.espresso.assertion.ViewAssertions
-import android.support.test.espresso.matcher.ViewMatchers
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.withId
 import com.schibsted.spain.barista.internal.matcher.DrawableMatchers
 
 object BaristaImageViewAssertions {
 
     @JvmStatic
     fun assertDrawable(@IdRes id: Int, @DrawableRes drawable: Int) {
-        Espresso.onView(ViewMatchers.withId(id)).check(ViewAssertions.matches(DrawableMatchers.withDrawable(drawable)))
+        onView(withId(id)).check(matches(DrawableMatchers.withDrawable(drawable)))
     }
 
+    @JvmStatic
+    fun assertDrawable(@IdRes id: Int) {
```

Comment:
Sorry for the late comment:
Now that we have multiple drawable assertions, I think the naming should be changed to explain the difference between each method:
- assertHasDrawable(id, drawable)
- assertHasAnyDrawable(id)
- assertHasNoDrawable(id)